test/functional/apps/visualize/group3/_pie_chart.ts

```diff
@@ -401,7 +401,10 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           ['360,000', '47', 'US', '4'],
           ['360,000', '47', 'BD', '3'],
           ['360,000', '47', 'BR', '2'],
-        ];
+        ].map((row) =>
+          // the count of records is not shown for every split level in the new charting library
+          isNewChartsLibraryEnabled ? [row[0], ...row.slice(2)] : row
```

Comment:
These changes are surprising to me - based on my understanding these tests were running for the new and the old implementation also in the old test configuration, right? Why are these issues only surfacing now?

Reply:
This is weird Joe, when I was implementing it I had checked and corrected the functional tests in order to work for both the old and the new implementation. Maybe something was broken with the unified renderers? 🤔

Reply:
Sorry, meant to leave a comment on the PR here. The tests  seem to fail to carry forward the Advanced Setting that enables the new charting library on main and we're always testing the legacy library. With the smaller config and less going on the tests work as expected do some assertions needed to be updated.

Reply:
Welp, glad we discovered this.